docs/sources/developers/plugins/variables.md

```diff
@@ -0,0 +1,60 @@
++++
+title = "Interpolate variables in plugins"
+type = "docs"
++++
+
+# Interpolate variables in plugins
+
+Variables are placeholders for values, and can be used to create things like templated queries. For more information on variables, refer to [Templates and variables]({{< relref "../../variables/templates-and-variables.md" >}}).
+
+This guide explains how to leverage template variables in your panel plugins and data source plugins.
+
+We'll see how you can turn a string like this:
+
+```sql
+SELECT * FROM services WHERE id = "$service"
+```
+
+into
+
+```sql
+SELECT * FROM services WHERE id = "auth-api"
+```
+
+Grafana provides a couple of helper functions to interpolate variables in a string template. Let's see how you can use them in your plugin.
+
+## Panels
+
+For panels, the [replaceVariables]({{< relref "../../packages_api/data/panelprops.md#replacevariables-property" >}}) function is available in the [PanelProps]({{< relref "../../packages_api/data/panelprops.md" >}}).
+
+Add `replaceVariables` to the argument list, and pass it a user-defined template string:
```

Comment:
```suggestion
Add `replaceVariables` to the argument list, and pass it a user-defined template string.
```

Reply:
As I commented in your other PR, you can go either way with the colons as long as you are consistent in your usage.